src/site/content/en/lighthouse-performance/unused-javascript/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Remove unused JavaScript
+description: |
+  Learn how to pass Lighthouse's "Remove unused JavaScript" audit.
+web_lighthouse:
+  - unused-javascript
+date: 2020-06-17
+---
+
+Unused JavaScript can slow down your page speed if the JavaScript is
+[render-blocking][crp]. When the browser encounters a render-blocking script
```

Comment:
The first sentence seems misplaced. Can the distinction of render-blocking scripts be moved further down?

It can slow down the page even if it isn't synchronous. unused async code competes with other resources for bandwidth, which can have significant performance implications.

Reply:
I'll resolve this convo and ping you to review the new text